spec/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "globals": {
+    "waitsForPromise": true
+  },
+  "env": {
+    "jasmine": true
+  }
+}
```

Comment:
I would suggest creating a few separate subdirectories inside spec/files, and putting the appropriate eslintrc files within these directories, each with "root:true" in them. Or, I can move things around in a separate PR.